.github/workflows/rule-validate.yml

```diff
@@ -123,7 +125,7 @@ jobs:
       # the latest commit, and we can depend on that as a required check.
       - name: "Create a check run"
         uses: actions/github-script@v6
-        if: github.event_name == 'pull_request_target' && ${{ always() }}
+        if: github.event_name == 'pull_request_target' && always()
```

Comment:
I tested this by expanding L5 to

```
on:
  push:
    branches: [ "main", "test-rules", "cd.test-comment ]
```

and it skipped this step like it should have (I didn't have a PR open on the branch): https://github.com/sublime-security/sublime-rules/actions/runs/7333566023/job/19969174991

BTW: `always()` in this context just means ignore previous step failures.